README.md

```diff
@@ -39,4 +44,4 @@ Or, if you don't have GraalVM installed, you can run the native executable build
 
 You can then execute your native executable with: `./target/t-learn-backend-1.0.0-SNAPSHOT-runner`
 
-If you want to learn more about building native executables, please consult https://quarkus.io/guides/building-native-image.
\ No newline at end of file
+If you want to learn more about building native executables, please consult https://quarkus.io/guides/building-native-image.
```

Comment:
what did change here?

Reply:
It's githubs weird way of showing that there is now an empty line at the end of the file